data-raw/vs_peds.R

```diff
@@ -183,11 +241,7 @@ vs_peds <- tibble::tribble(
     VSEVAL = NA_character_,
     EPOCH = "Epoch"
   ) %>%
-  select(
-    STUDYID, DOMAIN, USUBJID, VSSEQ, VSTESTCD, VSTEST,
-    VSORRES, VSORRESU, VSSTRESC, VSSTRESN, VSSTRESU, VSEVAL,
-    EPOCH, VISIT, VISITNUM, VSDTC, VSDY
-  )
+  arrange(USUBJID, VISITNUM, VSTESTCD, VSDY)
```

Comment:
no need to re sort at this stage - remove

Reply:
It has been removed.